comm.go

```diff
@@ -0,0 +1,61 @@
+package cleisthenes
+
+import (
+	"net"
+	"time"
+
+	"github.com/DE-labtory/cleisthenes/pb"
+	"google.golang.org/grpc"
+)
+
+type ConnHandler func(conn Connection)
+type ErrHandler func(err error)
+
+type GrpcServer struct {
+	connHandler ConnHandler
+	errHandler  ErrHandler
+	ip          string
+	lis         net.Listener
+	meta        map[string]string
+}
+
+func NewServer(meta map[string]string) *GrpcServer {
```

Comment:
what is `meta`?

Reply:
meta data of our grpc server, but maybe we don't need it now

I'll remove it